src/data/entities/PersistedConfig.ts

```diff
@@ -1,7 +1,11 @@
-import { Permission } from "../../domain/entities/Permission";
+import { Config } from "../../domain/entities/Config";
 
-export interface PersistedConfig {
+export interface PersistedConfig extends Config {
     poeditorToken?: string;
```

Comment:
I see that `poeditorToken` is nowhere in the code, so I would remove it and get rid of the "Persisted" concept